portal/viewer-release-notes.md

```diff
@@ -4,6 +4,17 @@
 The version of the viewer is shown in the Debug Info option.
 {% endhint %}
 
+## 4.12.33 - August 2022 - OHIF
+
+**The OHIF Viewer** is a zero-footprint medical image viewer provided by the [Open Health Imaging Foundation (OHIF)](http://ohif.org). It is a configurable and extensible progressive web application with out-of-the-box support for image archives which support [DICOMweb](https://www.dicomstandard.org/dicomweb/).
+
+Main highlights of this release include:
+
+* Fix parsing of qualitative slice annotation;
+* Disable measurements panel interactions in MPR mode;
+* Fix parsing of segmentation when orientation values are close to zero.
```

Comment:
@Punzo is this related to https://github.com/OHIF/Viewers/issues/2869?

Reply:
the third item yes! please do not merge this, because Steve wants to discuss https://github.com/dcmjs-org/dcmjs/pull/304 before merging it.

Reply:
Ok, then I am glad I caught it. But then would it still be in the 4.12.33 release? (I realized the title of this issue is incorrect, or the text in the actual changes is incorrect, since title states 4.12.32)

Reply:
yeap thanks for catching it! now it is actually 4.12.34. I would wait to merge, becauuse in case I will have a fix for https://github.com/OHIF/Viewers/issues/2849, the version will became 4.12.35 (: